api/lib/domain/read-models/participant-results/CompetenceResult.js

```diff
@@ -1,5 +1,5 @@
 class CompetenceResult {
-  constructor({ competence, area, totalSkillsCount, knowledgeElements, flashPixScore }) {
+  constructor({ competence, area, totalSkillsCount, knowledgeElements, flashPixScore, reachedStageForCompetence }) {
```

Comment:
On est dans `CompetenceResult` est-ce `reachedStage` ce n'est pas suffisant ? 

```suggestion
  constructor({ competence, area, totalSkillsCount, knowledgeElements, flashPixScore, reachedStage }) {
```

Reply:
Je comprends, je suis mitigée sur ce point. Sur le même principe que l'objet `ReachedStage` qui contient lui-même un `reachedStage`, on parle de 2 concepts métiers différents. Mais comme on le fait partout dans la codebase je vais appliquer le changement